examples_tests/49.ComputeFFT/convolve.comp

```diff
@@ -8,24 +8,21 @@
 
 layout(local_size_x=256, local_size_y=1, local_size_z=1) in;
  
-struct nbl_glsl_ext_FFT_output_t
-{
-	vec2 complex_value;
-};
+ #define complex_value vec2
```

Comment:
I see you use `complex_value` in multiple shaders, so its a good idea to have a `nbl_glsl_cvec2` and other definitions in `include/nbl/builtin/glsl/math/complex.glsl `

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/63#discussion_r572151748